File: Difficulty: Medium/Rotate Array/rotate-array.cpp
class Solution {
  public:
    void rotateArr(vector<int>& nums, int k) {
        // code here
                int n=nums.size();
            k=k%n;
        
        vector<int>num(n);
        for(int i=0;i<n;i++){
            num[i]=nums[(i+k)%n];
        }

            nums=num;
    }
};

File: Difficulty: Medium/Rotate Array/rotate-array.java
class Solution {
    public void rotateArr(int nums[], int k) {
        // code here
        int n=nums.length;
        int[] arr=nums.clone();
        
        for(int i=0;i<n;i++){
            nums[i]=arr[(i+k)%n];
        }
    }
}